doc/tor.md

```diff
@@ -86,7 +100,7 @@ and open port 8333 on your firewall (or use -upnp).
 If you only want to use Tor to reach onion addresses, but not use it as a proxy
 for normal IPv4/IPv6 communication, use:
 
-	./bitcoin -onion=127.0.0.1:9050 -externalip=57qr3yd1nyntf5k.onion -discover
+	./bitcoin -onion=127.0.0.1:9050 -externalip=57qr3yd1nyntf5k.onion -discover -listen -onlynet=onion -bind=127.0.0.1
```

Comment:
I think there is a slight misunderstanding here. There are two different cases:

1) Want to be able to connect to `.onion` peers, but normal IPv4/IPv6 connections go through plainnet
2) Want to be able to connect to `.onion` peers only

You change this from (1) to (2), but both are valid use cases, and I think if you make this change you should describe both.